### ISSUE TITLE
Policy to fix existing DeviceTypes definitions?

### ISSUE THREAD

**Issue by eduncan911** [netbox-community/devicetype-library#382]
What is the policy for submitting a PR(s) to fix deviceType definitions already checked into this repo?

Would it break some "remaining backwards compatible" rule in this repo?  Since basically people just import, I don't see it being a problem to change - as the way forward would have the correct definitions.

Last question: There are Ruckus devices listed under the Brocade folder.  Ruckus bought out Brocade, and all 7000 series devices are named Ruckus going forward.  However, they are listed under that Brocade folder.  

/tl;dr

I've recently come in possession of several Brocade and Ruckus (previously Brocade) networking gear.  

As I am trying to use Netbox devicetype-library imports, I have noticed that the vast majority are naming the ports and switch assignments completely wrong.

For example, I recently added the Ruckus ICX7150-C12P here: https://github.com/netbox-community/devicetype-library/pull/371

I spoke to the justification of the different naming conventions in that PR.  In short, all of the existing Brocade deviceTypes with `GigabitEthernet1/2` need to be changed to `ge-1/1/1`, `ge-1/1/2` and so on to match the switches' OS as well as Brocade's own literature.

Also, Netbox itself suggests `ge-1/1/1` - so that's a bonus.

However, I do not know the implications of making such changes to this repo for people with existing Netbox installations.

To be clear: I'm only talking about updating the Brocade/Ruckus deviceTypes that I am personally familiar with.  I would not touch any other types I do not know, nor any other manufacturer.

**Comment by ryanmerolle** [comment]
TLDR: Community corrections are  encouraged.

Additional Context:
Since this is all point in time imports and people can edit device definitions to their heart's content before importing, you should not worry about drift for people who have already imported.

We try our best to review PRs are accurate, but if we do not have personal experience with a platform, we will likely have these sort of issues.

**Comment by eduncan911** [comment]
Thank you!
